runtime/dotnet/azurewebapp/Schemas/update-schema.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Currently bfdialog has two bugs
1. On error it deletes the sdk.schema file and leaves the user with no schema file
2. It returns the 0 code despite error in merging schemas
Both these are in GH. To overcome this I take a backup of the file and revert to the backup if the bfdialog process does not generate a new sdk.schema file.

Unless they fix both these issues adding set -e might be risky for us. Let's say they fix just the error code bug. That still leaves our shell script in an unstable state since on error, the script exits immediately. This leaves the user with no .schema file as the portion of the script where I revert back to the original schema file will be skipped. Let me know if Im missing something here

Reply:
That makes sense. Are those issues with the CLI being tracked? Is that what you mean by "Both these are in GH"?

Reply:
Correct.
https://github.com/microsoft/botframework-cli/issues/806